test/Object/rlexe.xml

```diff
@@ -195,6 +195,13 @@
   </test>
   <test>
     <default>
+      <files>getOwnPropertyDescriptors.js</files>
+        <compile-flags>-args summary -endargs</compile-flags>
+        <tags></tags>
```

Comment:
You can omit the empty <tags>...</tags> line as it has the reasonable default of being empty. :)